gcc/rust/checks/errors/borrowck/ffi-polonius/src/gccrs_ffi.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright (C) 2020-2023 Free Software Foundation, Inc.
```

Comment:
ISTM you should only have 2023 for all the new files here, unless you are really reusing an existing file ?

Reply:
Oh, thanks for noticing. That is just bad copy-pasting, will fix.

Reply:
Fixed